components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -355,6 +359,27 @@ def validate_logs_dir(self):
         except ValueError as ex:
             raise ValueError(f"logs_directory is invalid: {ex}")
 
+    def load_execution_container_name(self):
+        if self.execution_container is not None:
```

Comment:
Just for sanity check, have you tested and made sure that the container specified in the clp-config will be properly picked up?

Reply:
Yes. I have updated the "Validation" section in the PR description to include the related instructions.